frontend/src/modules/staff/components/orders/types.ts

```diff
@@ -0,0 +1,78 @@
+export type OrderStatus =
+  | "PENDING"
+  | "IN_PROGRESS"
+  | "COMPLETED"
+  | "DELIVERED"
+  | "REJECTED";
+
+export interface BasicItem {
+  id: number;
+  name: string;
+  description?: string;
+  price: number;
+  type?: string;
+  status?: string;
+}
+
+export interface OrderItem {
+  id: number;
+  item: BasicItem;
+  orderId: number;
+  removed: boolean;
+}
+
+export interface ComboItem {
+  id: number;
+  combo: Combo;
+  item: BasicItem;
+  removed: boolean;
+}
+
+export interface Combo {
+  id: number;
+  name: string;
+  description?: string;
+  price: number;
+  status?: string;
+  comboItems: ComboItem[];
+}
+
+export interface OrderCombo {
+  comboId: number;
+  orderId: number;
+  combo: Combo;
+  removed: boolean;
+  orderComboItems: ComboItem[];
+}
+
+export interface User {
+  id: number;
+  username: string;
+  firstName?: string;
+  lastName?: string;
+  email?: string;
+}
+
+export interface Order {
+  id: number;
+  userId: number;
+  user: User;
+  status: OrderStatus;
+  totalPrice: number;
+  orderItems: OrderItem[];
+  orderCombos: OrderCombo[];
+  createdAt: Date;
+  updatedAt: Date;
+  removed: boolean;
+}
+
+export const statusColors: Record<
+  OrderStatus,
+  "default" | "primary" | "error" | "success" | "warning"
+> = {
```

Comment:
On the order status page on the user, there is a [list of colors](https://github.com/UPRMeals/UPRMeals/blob/836eafda5fd095e939f488d684ea7995e75c4208/frontend/src/pages/customers/orders/index.tsx#L20), what do you think of those? Feel free to leave these as is though. Also if you suggest that these are a better fit then we can refactor those as well for consistency :)

Reply:
No problem at all. Done